ldes-server-integration-test/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/LdesServerIntegrationTest.java

```diff
@@ -33,14 +33,12 @@
 		refresh = AutoConfigureEmbeddedDatabase.RefreshMode.AFTER_EACH_TEST_METHOD,
 		replace = AutoConfigureEmbeddedDatabase.Replace.ANY)
 @ActiveProfiles("postgres-test")
-@ContextConfiguration(classes = { MemberEntityRepository.class, PageRelationPostgresRepository.class, RelationEntityRepository.class })
+@ContextConfiguration(classes = { MemberEntityRepository.class, PageRelationPostgresRepository.class, PageRelationEntityRepository.class })
 @ComponentScan(value = { "be.vlaanderen.informatievlaanderen.ldes.server" })
 @TestPropertySource(properties = { "ldes-server.fragmentation-cron=*/1 * * * * *", "ldes-server.compaction-cron=*/10 * * * * *",
 		"ldes-server.deletion-cron=*/20 * * * * *", "ldes-server.compaction-duration=PT1S" })
 @SuppressWarnings("java:S2187")
 public class LdesServerIntegrationTest {
-	static final int FRAGMENTATION_POLLING_RATE = 1000;
```

Comment:
was going to come back to you for this. Since this is actually a too magic value that is so magic that it lost its value.. 
This was a quick fix to make tests fail quicker. But since you are more working on actual fragmentation, i did not want to touch these tests too much